pmd-core/src/main/java/net/sourceforge/pmd/lang/ast/Node.java

```diff
@@ -229,7 +230,25 @@ default Node getNthParent(int n) {
      * @param targetType class which you want to find.
      * @return List of all children of type targetType. Returns an empty list if none found.
      */
-    <T> List<T> findDescendantsOfType(Class<T> targetType);
+    default <T> List<T> findDescendantsOfType(Class<T> targetType) {
+        final List<T> list = new ArrayList<>();
+        TraversalUtils.findDescendantsOfType(this, targetType, list, false);
+        return list;
```

Comment:
this implementation seems bogus, you can simply call `return findDescendantsOfType(targetType, false);`